### ISSUE TITLE
Can you port this theme to zsh?

### ISSUE THREAD

**Issue by SCCOBN** [oh-my-fish/theme-bobthefish#298]
I had problems with fish displaying proper powerline characters, got ? instead of > graphic left of cursor, no guide online helped me fix it. Installing zsh and oh my zsh fixed all issues, so it's a problem with fish, yea fish sucks. 

Could you please port this theme for zsh?

**Comment by bobthecow** [comment]
It might be locale related. Several tickets have hinted at issues using non-UTF-8 locales with fish, e.g. https://github.com/oh-my-fish/theme-bobthefish/issues/125#issuecomment-640087544

I don't plan on porting bobthefish to zsh, because I don't use zsh. But you can feel free to 🙂

**Comment by SCCOBN** [comment]
Yes it was locale related I had to add to the fish file export LC_ALL=en_US=UTF-8.. strange since I have that uncommented in locale file.
